### DISCUSSION TITLE
Minecraft server?

### DISCUSSION THREAD

**Discussion by codergautam** [loophole/cli#101] [General]
Is it possible to make a minecraft server using loophole? Is there a tcp?

**Comment by 0x7f** [comment]
Hi, thanks for reaching out. I don't really know Minecraft, but from what I can tell from the [docs](https://picraft.readthedocs.io/en/release-0.6/protocol.html), the Minecraft protocol is a raw TCP protocol. Right now, loophole supports hosting http/https only. We have hosting of raw TCP services on our list, but right now, we do not support this and therefore also don't support Minecraft servers.

**Reply by codergautam** [reply]
F ok, thanks
